web/admin.py

```
@@ -770,3 +774,35 @@ class MembershipSubscriptionEventAdmin(admin.ModelAdmin):
+    list_filter = ("counter_type", "last_updated")
+    readonly_fields = ("last_updated",)
+
+
+@admin.register(UserActivity)
+class UserActivityAdmin(admin.ModelAdmin):
+    list_display = ("user", "activity_type", "get_content_object_str", "timestamp", "location")
+    list_filter = ("activity_type", "timestamp", "show_full_name")
+    search_fields = ("user__username", "location")
+    readonly_fields = ("timestamp",)
+
+    def get_content_object_str(self, obj):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_content_object_str`

(ANN201)